.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,15 +8,49 @@
 
 ## Summary ##
 
-<!-- Describe the change, including rationale and design decisions -->
+<!--
+Describe the change, including rationale and design decisions, and use a brief
+but descriptive PR title.
+
+The title should be prefixed with `[WIP]` before the checklist below is
```

Comment:
Otherwise it could be explicitly mentioned in the PR and we would add the label, if the person wants some feedback through the implementation